### ISSUE TITLE
WCAG / Pdf-Tagging

### ISSUE THREAD

**Issue by DmitriBountsman** [diegomura/react-pdf#3179]
I use this great libriary for several reports in the browser. The biggest issue for me is that WCAG is not supported at all even though pdfkit can handle the most accessibility requirements (https://pdfkit.org/docs/accessibility.html) like tagging, there is no way to add a structure to the document and its elements. And WCAG is getting required for many projects.

The feature  would be to:

1. extend Document to pass tagged (should be true by default for version > 1.3) and subset (should be 'PDF/UA' by default for for version > 1.3) attributes,
2. extend the components (especially View and Text) for required attributes like structType
3. adjust types for typescript (please add also bookmark for View and Text, missing at the moment )
4. add structure and markings to the document
5. create a test with different elements, passing complex structure like h1, h2, table...
6. test the created pdf-file with a tool like PAC (PDF Accessibility Checker), should show the tags/structure
7. fix the PDF-syntax-errors

**Comment by DmitriBountsman** [comment]
https://pdfa.org/resource/tagged-pdf-best-practice-guide-syntax/
